src/app/lib/streamer.js

```diff
@@ -324,7 +271,7 @@
                             info: torrent,
                             subtitle: model.get('subtitle'),
                             defaultSubtitle: model.get('defaultSubtitle'),
-                            title: title,
+                            title: model.get('title'),
```

Comment:
(I edited above)
Because 'title' is what will be used by loading.js (see: `streamInfo.set('title', torrent.title);`) to display an actual title, parsed, matched, and regexed, not just the basic title.

Your change makes no difference on providers giving a good title in the api directly, but it prevents dropped magnet/torrents from displaying the real title.